blocks/social/social.css

```diff
@@ -93,10 +111,14 @@ main .section.social-container .section-container ul {
 div.social-container h3 {
   font-weight: var(--font-weight-bold);
   font-size: var(--heading-font-size-ms);
-  margin-bottom: 1.5rem;
+  margin-bottom: 1rem;
 }
 
 @media (min-width: 62rem) {
+	main .section.social-container.narrow {
+    max-width: 54.5rem;
```

Comment:
I think `narrow` style should be consistent globally. Do we need to handle differently here?